packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -141,30 +153,50 @@
 />
 
 <aside class="on-this-page" bind:this={containerEl}>
-	<h2>On this page</h2>
-	<nav aria-label="On this page">
-		<ul>
-			<li>
-				<a
-					href="{base}/docs/{details.slug}"
-					class:active={hash === ''}
-					on:click={() => dispatch('select')}>{details.title}</a
-				>
-			</li>
-			{#each details.sections as { title, slug }}
+	<button class="heading" on:click={() => (mobile_menu_open = !mobile_menu_open)}>
```

Comment:
We should toggle `aria-expanded="true"` or `aria-expanded="false"` here

Reply:
Done